docs/docsite/rst/scenario_guides/vmware_scenarios/scenario_vmware_tools_connection.rst

```diff
@@ -0,0 +1,119 @@
+.. _vmware_tools_connection:
+
+************************************
+Using vmware_tools connection plugin
+************************************
+
+.. contents:: Topics
+
+Introduction
+============
+
+This guide will show you how to utilize VMware Connection plugin to communicate and automate various tasks on VMware guest machines.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.9 or later must be installed.
+
+    * We recommend installing the latest version with pip: ``pip install Pyvmomi`` on the Ansible control node
+      (as the OS packages are usually out of date and incompatible) if you are planning to use any existing VMware modules.
+
+* Hardware
+
+    * vCenter Server 6.5 and above
+
+* Access / Credentials
+
+    * Ansible (or the target server) must have network access to either the vCenter server
+
+    * Username and Password for vCenter with required permissions
+
+    * VMware tools or openvm-tools with required dependencies like Perl installed on the given virtual machine
+
+Caveats
+=======
+
+- All variable names and VMware object names are case sensitive.
+- You need to use Python 2.7.9 version in order to use ``validate_certs`` option, as this version is capable of changing the SSL verification behaviours.
+
+
+Example Description
+===================
+
+User can run playbooks against VMware virtual machines using ``vmware_tools`` connection plugin.
+
+In order work with ``vmware_tools`` connection plugin, you will need to specify hostvars for the given virtual machine.
+
+For example, if you want to run a playbook on a virtual machine called ``centos_7`` located at ``/Asia-Datacenter1/prod/centos_7`` in the given vCenter, you will need to specify hostvars like following:
+
+.. code-block:: ini
+
+    [centos7]
+    host1
+
+    [centos7:vars]
+    # vmware_tools related variables
+    ansible_connection=vmware_tools
+    ansible_vmware_host=10.65.201.128
+    ansible_vmware_user=administrator@vsphere.local
+    ansible_vmware_password=Esxi@123$%
+    ansible_vmware_validate_certs=no
+
+    # Location of the virtual machine
+    ansible_vmware_guest_path=Asia-Datacenter1/vm/prod/centos_7
+
+    # Credentials
+    ansible_vmware_tools_user=root
+    ansible_vmware_tools_password=Secret123
+
+Here, we are providing vCenter details and credentials for the given virtual machine to run playbook on.
+If your virtual machine path is ``Asia-Datacenter1/prod/centos_7``, you will specify ``ansible_vmware_guest_path`` as ``Asia-Datacenter1/vm/prod/centos_7``. Please take a note that ``/vm`` is added in the virtual machine path, since this is a logical folder structure in VMware inventory.
```

Comment:
```suggestion
If your virtual machine path is ``Asia-Datacenter1/prod/centos_7``, you specify ``ansible_vmware_guest_path`` as ``Asia-Datacenter1/vm/prod/centos_7``. Please take a note that ``/vm`` is added in the virtual machine path, since this is a logical folder structure in the VMware inventory.
```